data/css/modules/contentGroup/_contentGroup.scss

```diff
@@ -94,6 +94,17 @@
             -gtk-icon-shadow: none;
         }
     }
+
+    &__paginate {
+        background-color: $brand-color;
```

Comment:
See other comment; other than this, the two paginate commits are ready to go to master.